specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-01-preview/machineLearningServices.json

```diff
@@ -0,0 +1,5247 @@
+        },
+        "name": {
+          "type": "string",
+          "description": "Friendly name of the linked workspace."
+        },
+        "type": {
+          "type": "string",
+          "description": "Resource type of linked workspace."
+        },
+        "properties": {
+          "$ref": "#/definitions/LinkedWorkspaceProps"
+        }
+      }
+    },
+    "LinkedWorkspaceDto": {
+      "type": "object",
+      "description": "object used for creating linked workspace.",
+      "properties": {
+        "name": {
+          "description": "Friendly name of the linked workspace",
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/LinkedWorkspaceProps"
+        }
+      }
+    },
+    "LinkedWorkspaceProps": {
+      "type": "object",
+      "description": "LinkedWorkspace specific properties.",
+      "properties": {
+        "linkedWorkspaceResourceId": {
+          "description": "ResourceId of the link target of the linked workspace.",
+          "type": "string"
+        },
+        "userAssignedIdentityResourceId": {
+          "description": "ResourceId of the user assigned identity for the linked workspace.",
+          "type": "string"
+        }
+      }
+    },
+    "ServiceResponseBase": {
+      "description": "The base service response. The correct inherited response based on computeType will be returned (ex. ACIServiceResponse)",
+      "required": [
+        "computeType"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The service Id.",
+          "type": "string"
+        },
+        "name": {
+          "description": "The service name.",
+          "type": "string"
+        },
+        "description": {
+          "description": "The service description.",
+          "type": "string"
+        },
+        "kvTags": {
+          "description": "The service tag dictionary. Tags are mutable.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "properties": {
+          "description": "The service property dictionary. Properties are immutable.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "operationId": {
+          "description": "The ID of the latest asynchronous operation for this service.",
+          "type": "string"
+        },
+        "state": {
+          "description": "The current state of the service.",
+          "enum": [
+            "Transitioning",
+            "Healthy",
+            "Unhealthy",
+            "Failed",
+            "Unschedulable"
+          ],
+          "type": "string",
+          "example": "Healthy",
+          "x-ms-enum": {
+            "name": "WebServiceState",
+            "modelAsString": false
+          }
+        },
+        "createdTime": {
```

Comment:
If you need createdAt/By, etc... info please implement the top-level systemData contract https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources

Reply:
will make those changes  in our next preview version.